src/test/kotlin/reactor/kotlin/test/StepVerifierExtensionsTests.kt

```diff
@@ -53,4 +58,49 @@ class StepVerifierExtensionsTests {
                 .verifyError<IllegalStateException>()
     }
 
+    @Test
+    fun `testUsingVirtualTime()`() {
+        { Mono.just("foo").delayElement(Duration.ofDays(1)) }
+            .testUsingVirtualTime()
+            .thenAwait(Duration.ofDays(1))
+            .expectNext("foo")
+            .verifyComplete()
```

Comment:
alternatively, each of these tests could use the `.expectComplete().verify(Duration.ofSeconds(1))` style here, which would put a timeout on the whole verification

Reply:
I think that's the better approach, so I decided to go with that one. As a bonus, the code is now indented in the same style as Sergei's tests above :)